code/modules/customitems/item_defines.dm

```diff
@@ -1578,4 +1578,143 @@ All custom items with worn sprites must follow the contained sprite system: http
 	icon = 'icons/obj/custom_items/jennifer_clothes.dmi'
 	icon_state = "jennifer_shoes"
 	item_state = "jennifer_shoes"
-	contained_sprite = TRUE
\ No newline at end of file
+	contained_sprite = TRUE
+
+
+/obj/item/clothing/suit/storage/toggle/fluff/blessing_jacket //Armored Detective Jacket - Nelson Blessing - seniorscore
+	name = "armored detective jacket"
+	desc = "A white suit jacket, has a badge hanging out of a breast pocket. Touching it gives a feeling of working on a case for months."
+	icon = 'icons/obj/custom_items/blessing_jacket.dmi'
+	icon_state = "blessing_jacket"
+	item_state = "blessing_jacket"
+	icon_open = "blessing_jacket_open"
+	icon_closed = "blessing_jacket"
+	contained_sprite = TRUE
+
+
+/obj/item/clothing/under/dress/fluff/katya_dress //Tailored Tajara Dress - Katya Al-Tahara - coalf
+	name = "tailored tajaran dress"
+	desc = "A simple long, blue and flowing dress, it has a knitted overthrow that fits over the shoulder and arms."
+	icon = 'icons/obj/custom_items/katya_clothing.dmi'
+	icon_state = "katya_dress"
+	item_state = "katya_dress"
+	contained_sprite = TRUE
+
+/obj/item/clothing/under/fluff/katya_uniform //Messy Work Clothes - Katya Al-Tahara - coalf
+	name = "messy work clothes"
+	desc = "A simple pants and shirt combo, the white shirt has long since faded and are those crumbs?"

```

Comment:
Grammar's a little dicey here. Should be some kind of pause or connective around `faded` and `and`.